plugins/InspectionGadgets/InspectionGadgetsAnalysis/src/com/siyeh/InspectionGadgetsBundle.properties

```diff
@@ -1241,7 +1241,8 @@ string.equals.empty.string.option.do.not.add.null.check=Do not report when null-
 tail.recursion.replace.quickfix=Replace tail recursion with iteration
 if.statement.with.too.many.branches.max.option=Maximum number of branches:
 if.statement.with.too.many.branches.problem.descriptor=<code>#ref</code> has too many branches ({0}) #loc
-negated.conditional.ignore.option=Ignore '!= null' comparisons
+negated.conditional.ignore.null.option=Ignore '!= null' comparisons
```

Comment:
Could you just reuse the two identical "negated.if.else..." properties below, and remove the two "negated.conditional..." properties?